content/docs/details/scaffolder/index.md

```diff
@@ -47,6 +49,12 @@ spec:
         message: 'Hello, ${{ parameters.name }}!'
 ```
 
+## Template preview
```

Comment:
Is it true that the previewer can only preview the UI created by the "parameters" section of the website?

If that is the case, perhaps this content belongs down under the parameters heading, further down the document?

I suggest the heading "Previewing parameters", and placing it [here](https://deploy-preview-756--roadie.netlify.app/docs/details/scaffolder/#more-reading).

Reply:
Yes, it makes sense, will add